DESCRIPTION

```diff
@@ -25,7 +25,7 @@ Depends:
 Imports:
     methods,
     IRanges,
-    MsCoreUtils (>= 1.1.1),
+    MsCoreUtils (>= 1.1.4),
```

Comment:
I just pushed `MsCoreUtils` 1.1.**3** to bioc. I decrement the version number in master already.